src/packages/core/property-editor/uis/tree-picker/manifests.ts

```diff
@@ -15,9 +15,19 @@ const manifest: ManifestPropertyEditorUi = {
 			properties: [
 				{
 					alias: 'startNode',
-					label: 'Start node',
+					label: 'Node type',
 					description: '',
 					propertyEditorUiAlias: 'Umb.PropertyEditorUi.TreePicker.StartNode',
+					config: [
```

Comment:
If we change the input-start-node to input-content-type then I think we can hardcode these values inside the input element instead. Then we don't have to pass this config value all the way down to the input element. It might simplify the code a little bit.